### DISCUSSION TITLE
Jenkins template engine

### DISCUSSION THREAD

**Discussion by arjunjte** [community/community#119448] [Programming Help]
### Select Topic Area

General

### Body

Hi team. I have configured Jenkins template engine in Jenkins. 
I have a library repository structure like libraries/npm/steps/complice.groovy and libraries/common/steps/code.groovy.
And also Jenkins file having in library repository itself like pipeline-config/Jenkinsfile
how to import code.groovy library in libraries/npm/steps/complice.groovy.
Application code repository having config file like(file name: pipeline_config.groovy)
-----------------------------------------------------------------------------------------------------------------------
libraries{
npm
common
} 

I tryed mutiple ways but I am not able to importing code.groovy file library into complice.groovy library.
can any one help me on this issue.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @arjunjte !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Code Search and Navigation category is solely related to conversations around the GitHub product CS&N. This question would be best received in the Programming Help category. I've gone ahead and moved it for you. Good luck!
